renku/core/models/workflow/parameters.py

```diff
@@ -265,6 +265,53 @@ def as_jsonld(self):
         return CommandOutputSchema().dump(self)
 
 
+@attr.s(cmp=False,)
+class RunParameter:
+    """A run parameter that is set inside the script."""
+
+    _id = attr.ib(default=None, kw_only=True)
+
+    _label = attr.ib(default=None, kw_only=True)
+
+    name = attr.ib(default=None, type=str, kw_only=True)
+
+    value = attr.ib(default=None, type=str, kw_only=True)
+
+    type = attr.ib(default=None, type=str, kw_only=True)
+
+    @staticmethod
+    def generate_id(run_id, name):
+        """Generate an id."""
+        name = urllib.parse.quote(name, safe="")
+        return "{}/arguments/{}".format(run_id, name)
```

Comment:
should we change the id to `[...]/parameter/[...]` ?

Reply:
Was waiting for a second thought on that 😄